src/main/java/org/broadinstitute/consent/http/models/DarCollectionSummary.java

```diff
@@ -43,6 +43,8 @@ public class DarCollectionSummary {
   @JsonProperty
   private int datasetCount;
 
+  private Integer researcherId;
```

Comment:
I used `researcherId` to match `researcherName`, but I can alternatively use `createUserId` since that's normally what it is referred to as